__tests__/nc-news.test.js

```diff
@@ -364,4 +365,102 @@ describe("app", () => {
           })
       })
     })
-});
\ No newline at end of file
+  describe("GET - /api/articles(queries)", () => {
+    it("200: GET - should accept a topic query and respond with an array of articles filtered by the topic value specified. if the topic is missing, it should respond with all articles", () => {
+      
+      return request(app)
+        .get("/api/articles?topic=mitch")
+        .expect(200)
+        .then(({ body }) => {
+          const {articles} = body;
+
+          expect(body).toHaveProperty("articles");
+          expect(articles).toBeInstanceOf(Array)
+          expect(articles).toHaveLength(11);
+          articles.forEach(article => {
+            expect(article).toHaveProperty("topic", "mitch")
+          })
+        });
+    })
+    it("200: GET - should accept a topic query and a sort_by query respond with an array of articles(filtered by the topic value if present) sorted by any valid column (defaults to date)", () => {
+      
+      return request(app)
+        .get("/api/articles?topic=mitch&sort_by=author")
+        .expect(200)
+        .then(({ body: {articles} }) => {
+
+          expect(articles).toBeSortedBy("author", { descending: true })
```

Comment:
I think this might default to true if the articles length=0. Know you've asserted against the length in the previous test of the same endpoint but may be worth having in the same test block in the future

Reply:
I think this might default to true if the articles length=0. Know you've asserted against the length in the previous test of the same endpoint but may be worth having in the same test block in the future

Reply:
Thanks Joe, I have added an expect line in the test block just in case! :)